tests/model_indexes/tests.py

```diff
@@ -62,6 +73,13 @@ def test_condition_requires_index_name(self):
         with self.assertRaisesMessage(ValueError, 'An index must be named to use condition.'):
             models.Index(condition=Q(pages__gt=400))
 
+    def test_fields_expressions_requires_index_name(self):
+        with self.assertRaisesMessage(
```

Comment:
It seems that the convention is to use a `msg`-variable and keeping `assertRaisesMessage` on one line.